Eat_and_Greet/src/screens/OnboardingScreens/emailVerify.tsx

```diff
@@ -4,10 +4,38 @@ import { useNavigation } from '@react-navigation/native';
 import styles from '../../style';
 import RoundedButton from '../../comps/RoundedButton/RoundedButton';
 import React from 'react';
+import { sendEmailVerification, getAuth } from 'firebase/auth';
 
 function EmailVerify() {
     const navigation = useNavigation();
     
+    const auth = getAuth();
+
+    const emailVerify = async () => {
+        console.log("in email verify");
+        try {
+            if (auth.currentUser && !auth.currentUser?.emailVerified) {
+                await sendEmailVerification(auth.currentUser);         
+                alert("Email Sent!");   
+            } else {
+                alert("Email already verified!");
+            }
+        } catch (e) {
+            console.log(e);
```

Comment:
Do we want this console logged or put in a log file? Console log only helps us debug if it's done by us, whereas if we want to check otherwise we'd want a log. Worth considering but nonessential.